scripts/tag_west_repos.sh

```diff
@@ -36,20 +36,27 @@
 
 declare -A PROJECT_TAGS
 
-# Uncomment and edit these lines as needed.
+# Set the tag names for all the projects below.
 #
 #     - keys are project names in west.yml
 #     - values are the tags you want to make
 #
-# PROJECT_TAGS[zephyr]=v2.7.99-ncs1-rc1
-# PROJECT_TAGS[mcuboot]=v1.8.99-ncs1-rc1
-# PROJECT_TAGS[trusted-firmware-m]=v1.5.0-ncs1-rc1
-# PROJECT_TAGS[matter]=v1.9.0-rc1
-# PROJECT_TAGS[openthread]=v1.9.0-rc1
-# PROJECT_TAGS[nrfxlib]=v1.9.0-rc1
-# PROJECT_TAGS[find-my]=v1.9.0-rc1
-# PROJECT_TAGS[nrf-802154]=v1.9.0-rc1
-# PROJECT_TAGS[homekit]=v1.9.0-rc1
+# nrfx/nRF/srcx repositories: vX.Y.Z(-rcN)
+PROJECT_TAGS[nrfxlib]=""
+PROJECT_TAGS[find-my]=""
+PROJECT_TAGS[homekit]=""
+PROJECT_TAGS[matter]=""
+PROJECT_TAGS[nrf-802154]=""
+
+# OSS repositories: vX.Y.Z-ncsN(-rcM)(-I)
+PROJECT_TAGS[zephyr]=""
+PROJECT_TAGS[mcuboot]=""
+PROJECT_TAGS[trusted-firmware-m]=""
+PROJECT_TAGS[tfm-mcuboot]=""
+PROJECT_TAGS[openthread]=""
+PROJECT_TAGS[mbedtls]=""
```

Comment:
Isn't `mbedtls` coming from Zephyr? Sorry if I mix up things, it's a bit confusing to figure what mbedtls instance comes from where.

Reply:
Yes, this one comes from Zephyr, but Zephyr's fork also has a corresponding upstream version (3.0.0 right now).

Reply:
Ok, I still don't understand what do we do with the tag - we tag the commit we use with let's say `v3.0.0-ncs1` and then what - we don't have a place to push it (we're not going to push the tag to https://github.com/zephyrproject-rtos/mbedtls, no?)

Reply:
we actually have this fork: https://github.com/nrfconnect/sdk-mbedtls/

Reply:
But that's `mbedtls-nrf`, no?

Reply:
Not according tho this:

![image](https://user-images.githubusercontent.com/9802583/153572132-365e18b0-1bd4-42b1-ac5c-8d00c7391600.png)

Reply:
> Not according tho this:
> 
> ![image](https://user-images.githubusercontent.com/9802583/153572132-365e18b0-1bd4-42b1-ac5c-8d00c7391600.png)

I'm not sure how does that contradicts with what I wrote above, `sdk-mbedtls` is a repository which is used by the module called `mbedtls-nrf` in our manifest. Here we're discussin module named `mbedtls` which is inherited from Zephyr.

Reply:
Yes, you are right, unfortunately we cannot tag mbedtls, which would be great because it's an often-repeated question from our users. I have updated the PR now.

Reply:
For the future, if `sdk-mbedtls` is compatible with the upstream, we could treat it as ony other OSS repo and just keep a single mbedTLS instance in the tree? It'd make sense then though to keep it in the same location as Zephyr.